web-client/src/transaction.rs

```diff
@@ -320,3 +335,229 @@ impl Transaction {
         &self.inner
     }
 }
+
+#[derive(serde::Serialize, serde::Deserialize)]
+pub struct Raw {
+    raw: String,
+}
```

Comment:
Maybe:
```suggestion
pub struct Raw(String);
```
?

Reply:
The format is supposed to be `{ raw: string }` in JS, so I think what I have is correct. This is the structure used by the 1.0 library.